appshell/cefclient.cpp

```diff
@@ -32,6 +33,20 @@ CefWindowHandle AppGetMainHwnd() {
   return g_handler->GetMainHwnd();
 }
 
+// CefCommandLine::HasSwitch is unable to report the presense of switches,
```

Comment:
i'm not too sure about this. can you verify with "force-renderer-accessibility" without the leading "--" and use the existing method.

Reply:
@vickramdhawal I tried doing searching for the switch without the — . It is not working. However when I search for the same switch in ‘ClientApp::OnBeforeCommandLineProcessing’ it is being found. Again the same is not found in ‘ClientApp::OnBeforeChildProcessLaunch’. Something weird going on there.

Reply:
If this is confirmed, then the changes LGTM :+1